assets/server/realmadmin/_stats_keyserver.html

```diff
@@ -295,6 +297,9 @@ <h5 class="modal-title">Onset-to-upload distribution</h5>
     dataTable.addColumn('number', 'Total');
     dataTable.addColumn({type:'string', role:'tooltip', p: {html: true}});
     dataTable.addColumn('number', 'Missing Onset Date');
+    {{if $realm.AllowsUserReport}}
```

Comment:
I'd really prefer if we did this at the javascript layer

Reply:
done, although it makes the JS more complicated

Reply:
Yea, it just enables us to move the JS into a CDN in the future 😄